pkg/inspector/image-inspector.go

```diff
@@ -232,61 +77,66 @@ func main() {
 			}
 		}
 		if authErr != nil {
-			log.Fatalf("Unable to pull docker image: %v\n", authErr)
+			return fmt.Errorf("Unable to pull docker image: %v\n", authErr)
 		}
 	} else {
-		log.Printf("Image %s is available, skipping image pull", inspectorOptions.Image)
+		log.Printf("Image %s is available, skipping image pull", i.opts.Image)
 	}
 
 	// For security purpose we don't define any entrypoint and command
+	randomName, err := generateRandomName()
```

Comment:
@pweil- now the comment above is misplaced, it should be close to `container, err := client.CreateContainer(...`